.circleci/scripts/git-diff-develop.ts

```diff
@@ -76,14 +96,31 @@ async function gitDiff(): Promise<string> {
  */
 async function storeGitDiffOutput() {
   try {
-    console.log("Attempting to get git diff...");
-    const diffOutput = await gitDiff();
-    console.log(diffOutput);
-
     // Create the directory
+    // This is done first because our CirleCI config requires that this directory is present,
+    // even if we want to skip this step.
     const outputDir = 'changed-files';
     fs.mkdirSync(outputDir, { recursive: true });
 
+    console.log(`Determining whether this run is for a PR targetting ${MAIN_BRANCH}`)
+    if (!process.env.CIRCLE_PULL_REQUEST) {
+      console.log("Not a PR, skipping git diff");
```

Comment:
I don't have enough context on how this works. How can this not be a PR?

Reply:
CircleCI can be triggered for branches without PRs. And it can be triggered for tags as well. In those cases, this environment variable would be missing.

Reply:
🙇